Scenes/HUD/WaveStatus.tscn

```diff
@@ -4,7 +4,6 @@
 [ext_resource type="Script" path="res://Scenes/HUD/WaveStatus.gd" id="2_fsc15"]
 
 [node name="WaveStatus" type="VBoxContainer" node_paths=PackedStringArray("_label", "_stats_label", "_timer_label")]
-custom_minimum_size = Vector2(650, 0)
```

Comment:
This overwrites a change from main branch.

Reply:
Restored